backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/webservice/schema/types/CardVerificationModel.kt

```diff
@@ -2,5 +2,5 @@ package app.ehrenamtskarte.backend.verification.webservice.schema.types
 
 data class CardVerificationModel(
     val cardInfoHashBase64: String,
-    val totp: Int
+    val totp: Int?
```

Comment:
Can you add in CodeType here as well (and the validation check in the query endpoint) to make the query explicit as well?

Reply:
I guess this will also be resolved in the frontend pr?